src/canopy/llm/cohere.py

```diff
@@ -210,6 +212,44 @@ async def agenerate_queries(self,
                                 ) -> List[Query]:
         raise NotImplementedError("Cohere LLM doesn't support async query generation")
 
+    def _convert_model_params(self, openai_model_params: dict) -> dict:
+        """
+        Convert Open AI model params to Cohere equivalents.
+
+        Args:
+            openai_model_params: model params passed from client to Canopy API in OpenAI format.
+
+        Returns:
+            Model params used with Cohere Chat API.
+        """  # noqa: E501
+        common_params = [
```

Comment:
As discussed, larger surface area can be covered here with https://cohere-sdk.readthedocs.io/en/latest/cohere.html#cohere.client.Client.chat params

Reply:
@giannis2two I have updated this method to handle a few of the parameters that were previously undocumented.

I have added `logit_bias` to the `common_params` list, and I have added `top_p` and `user` and mapped them to the Cohere Co.Chat API's `p` and `user_name` parameters.